packages/cli/src/cli.ts

```diff
@@ -28,6 +28,7 @@ import { YargsCheckError } from "./YargsCheckError.js";
 
 export async function cli(args: string[] = process.argv) {
   const base: Argv<CliCommonArgs> = yargs(hideBin(args))
+    .wrap(Math.min(150, process.stdout.columns))
```

Comment:
Should we use yargs built in method for this?

```suggestion
    .wrap(Math.min(150, yargs().terminalWidth()))
```